<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEditRangeUIntWidget</class>
 <widget class="QWidget" name="CEditRangeUIntWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSpinBox" name="startSpinBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>99998</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="endSpinBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="currentSpinBox">
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="object_viewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>currentSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
